src/builder/modules/Generator_Builder.res

```diff
@@ -437,6 +437,51 @@ let fillBackgroundColor = (model: Model.t, fillStyle: string) => {
   }
 }
 
+let drawLine = (
+  model: Model.t,
+  (x0, y0): position,
+  (x, y): position,
```

Comment:
Minor: Let's name these `(x1, y1)` and `(x2, y2)` just to match the common notation for coordinates in 2D.